tests/static-volume_test.go

```diff
@@ -89,7 +89,11 @@ var _ = Describe("checkStaticVolume tests", func() {
 			Expect(err).ToNot(HaveOccurred())
 			pv.Spec.PersistentVolumeReclaimPolicy = corev1.PersistentVolumeReclaimRetain
 			_, err = f.K8sClient.CoreV1().PersistentVolumes().Update(context.TODO(), pv, metav1.UpdateOptions{})
-			Expect(err).ToNot(HaveOccurred())
+			// We shouldn't make the test fail if there's a conflict with the update request.
+			// These errors are usually transient and should be fixed in subsequent retries.
```

Comment:
I don't think we will retry over this failing with `errors.IsConflict`, unless I'm missing something

Reply:
If this fails with `IsConflict` it'll then retry the update in the `Eventually` below, but won't have the updated reclaim policy. I'll fix it.